### PR TITLE
Fix `recently-pushed-branches-enhancements` on GHE

### PR DESCRIPTION
**Pull request by busches**
GHE still uses the old fixed with header, along with different markup/classes.

### CODE REVIEW

**Comment by busches** [comment]
And found it was duplicated on a page still, one more fix incoming.

File: source/features/recently-pushed-branches-enhancements.tsx [hunk 1]
```diff
@@ -54,7 +54,7 @@ async function init(): Promise<false | void> {
 	document.body.classList.add('rgh-recently-pushed-branches');
 
 	// Move or add list next to the notifications bell
-	select('.Header-item--full')!.after(widget);
+	select('.Header-item--full,.HeaderMenu nav')!.after(widget);
```
Comment (bfred-it) [inline source/features/recently-pushed-branches-enhancements.tsx]: `HeaderMenu` doesn't exist on regular GitHub, so no chances of conflict. If this works for you we can merge it

**Comment by busches** [comment]
@bfred-it I'm running into this: 
```css
.rgh-recently-pushed-branches [data-url$='recently_touched_branches_list'] p {
	display: none;
}
```
not hiding the original box, how does this not hide both once we move it?

**Comment by busches** [comment]
Actually, now I can't reproduce it, so 🤷‍♂️

**Pull request merged**